File: DotNet/Node.Core/Biz/Interfaces/Submit/IPostProcess.cs
using System;

using Node.Core.Biz.Manageable.Parameters;
using Node.Core.Document;

namespace Node.Core.Biz.Interfaces.Submit
{
    /// <summary>
    /// The Interface for Submit PostProcess
    /// </summary>
    public interface IPostProcess
    {
        /// <summary>
        /// Process of Submit.
        /// </summary>
        /// <param name="token">The Token which is result of Authentication Operation.</param>
        /// <param name="transID">The transaction id is result of Solicict/Submiet Operation.</param>
        /// <param name="dataFlow">Specifies which dataflow are to be used.</param>
        /// <param name="docs">Array of files being submitted</param>
        /// <param name="param">Operation related parameters</param>
        /// <returns>Transaction ID</returns>
        void Execute(string token, string transID, string dataFlow, NodeDocument[] docs, PostParam param);
    }
}

File: EN Node for Java environment/Node/src/Node/Biz/Interfaces/Submit/IPostProcess.java
package Node.Biz.Interfaces.Submit;

import java.rmi.RemoteException;

import Node.Biz.Custom.PostParam;
import Node.WebServices.Document.ClsNodeDocument;
/**
 * <p>This class create Submit Post Process Interface.</p>
 * <p>Company: enfoTech & Consulting, Inc.</p>
 * @author enfoTech
 * @version 2.0
 */

public interface IPostProcess {
	  /**
	   * Execute
	   * @param token The authentication token
	   * @param transID The transaction ID
	   * @param dataFlow The data flow name
	   * @param docs The submit file array
	   * @param param PostParam object
	   * @throws RemoteException
	   * @return PostParam object
	   */
  public PostParam Execute (String token, String transID, String dataFlow, ClsNodeDocument[] docs, PostParam param) throws RemoteException;
}
